ansible/roles/ghostunnel/defaults/main.yml

```diff
@@ -0,0 +1,225 @@
+# Basic configuration [[[
+# -----------------------
+
+# .. envvar:: ghostunnel__user [[[
+#
+# Name of the UNIX system account which is used to run Ghostunnel.
+ghostunnel__user: 'ghostunnel'
+
+                                                                   # ]]]
+# .. envvar:: ghostunnel__group [[[
+#
+# Name of the UNIX system group which is used to run Ghostunnel.
+ghostunnel__group: 'ghostunnel'
+
+                                                                   # ]]]
+# .. envvar:: ghostunnel__default_append_groups [[[
+#
+# List of default additional system groups to append to the Ghostunnel system user.
+ghostunnel__default_append_groups:
+
+  - 'ssl-cert'
+
+                                                                   # ]]]
+# .. envvar:: ghostunnel__append_groups [[[
+#
+# List of additional system groups to append to the Ghostunnel system user.
+# Some groups are required for access to sockets.
+ghostunnel__append_groups: []
+
+                                                                   # ]]]
+# .. envvar:: ghostunnel__combined_append_groups [[[
+#
+# The variable which combines all other Ghostunnel additional system groups
+# to append to the Ghostunnel system user and is used in the role tasks and templates.
+ghostunnel__combined_append_groups: '{{ ghostunnel__default_append_groups
+                                        + ghostunnel__append_groups }}'
+
+                                                                   # ]]]
+# .. envvar:: ghostunnel__services [[[
+#
+# Ghostunnel services to configure.
+# :ref:`ghostunnel__services` for more details.
+ghostunnel__services: []
+
+                                                                   # ]]]
+# .. envvar:: ghostunnel__group_services [[[
+#
+# Ghostunnel services to configure in specific Ansible
+# inventory group.
+ghostunnel__group_services: []
+
+                                                                   # ]]]
+# .. envvar:: ghostunnel__host_services [[[
+#
+# Ghostunnel services to configure in specific Ansible
+# inventory host.
+ghostunnel__host_services: []
+
+                                                                   # ]]]
+# .. envvar:: ghostunnel__dependent_services [[[
+#
+# Ghostunnel services configured by other Ansible roles using role
+# dependent variables.
+ghostunnel__dependent_services: []
+
+                                                                   # ]]]
+# .. envvar:: ghostunnel__combined_services [[[
+#
+# The variable which combines all other Ghostunnel services
+# configuration variables and is used in the role tasks and templates.
+ghostunnel__combined_services: '{{ ghostunnel__services
+                                   + ghostunnel__group_services
+                                   + ghostunnel__host_services
+                                   + ghostunnel__dependent_services }}'
+
+                                                                   # ]]]
```

Comment:
Use two fold markers here for proper section holding, don't leave a blank line between the variable's value and fold markers at the end of the section.

Reply:
Done!